pkg/types/metrics.go

```diff
@@ -127,6 +128,7 @@ func (o *OTLP) SetDefaults() {
 	o.AddServicesLabels = true
 	o.ExplicitBoundaries = []float64{.005, .01, .025, .05, .075, .1, .25, .5, .75, 1, 2.5, 5, 7.5, 10}
 	o.PushInterval = types.Duration(10 * time.Second)
+	o.ServiceName = "traefik"
```

Comment:
It would be great to have a test when overriding this value. There is already a test called `TestOpenTelemetry` which tests the default values. Either you add a new test based on this one, or you could add a test case to test overriding the `ServiceName`.